packages/angular_devkit/build_optimizer/src/build-optimizer/webpack-loader.ts

```diff
@@ -17,9 +17,27 @@ interface BuildOptimizerLoaderOptions {
   sourceMap: boolean;
 }
 
+export const buildOptimizerLoaderPath = __filename;
+
 export default function buildOptimizerLoader
   (this: webpack.loader.LoaderContext, content: string, previousSourceMap: RawSourceMap) {
   this.cacheable();
+
+  const skipBuildOptimizer = this._module
+    && this._module.factoryMeta
+    && this._module.factoryMeta.skipBuildOptimizer;
+
+  if (skipBuildOptimizer) {
+    // Skip loading processing this file with Build Optimizer if we determined in
+    // BuildOptimizerWebpackPlugin that we
```

Comment:
Nit: seems like the comment is unfinished

Reply:
Good catch 👍